service/build.gradle

```diff
@@ -44,6 +45,10 @@ dependencies {
     implementation 'org.springframework.boot:spring-boot-starter-validation'
     implementation 'org.springframework.boot:spring-boot-starter-thymeleaf'
 
+    // https://projectlombok.org
+    compileOnly 'org.projectlombok:lombok:1.18.20'
```

Comment:
added lombok to cut down on the entity boilerplate, can see the difference [here](https://github.com/DataBiosphere/terra-scientific-pipelines-service/pull/32/commits/75e2a4ad8fa17fc12d6b547b33d157a7dcd151d7).  Happy to take it out if we dont think its worth it.